opentelemetry-sdk/src/opentelemetry/sdk/trace/__init__.py

```diff
@@ -333,6 +334,13 @@ def __init__(
         self._current_span_slot = Context.register_slot(slot_name)
         self._active_span_processor = MultiSpanProcessor()
         self.sampler = sampler
+        self._atexit_hanlder = atexit.register(
```

Comment:
My personal preference would be to **not** do this by default, but have it as something the user can plug himself, or at least make this optional.

Reply:
@carlosalberto why not do this by default? Because it would be surprising that calling `del` results in (blocking!) network calls?

I agree about making this configurable, but I think this is much better than the current default: call `shutdown` or lose unexported spanns.